backend/application/syson-sysml-import/src/main/java/org/eclipse/syson/sysml/parser/AstTreeParser.java

```diff
@@ -188,10 +188,10 @@ public void resolveAllReference(List<ProxiedReference> unresolvedReferences0) {
     private void printResolutionError(List<ProxiedReference> proxiedReferences) {
 
         for (ProxiedReference pr : proxiedReferences) {
-            String msg = MessageFormat.format("[Unable to resolve following proxy] Owner ''{0}'' to qualifiedName ''{1}'' for reference ''{2}''", this.logNameProvider.getName(pr.owner()),
+            String msg = MessageFormat.format("Unable to resolve a reference ''{1}'' on element ''{0}'' for reference ''{2}''", this.logNameProvider.getName(pr.owner()),
```

Comment:
Very strange message, please fix it, especially the fact the reference is mentioned twice.

Reply:
Done.